roboy_dialog/src/main/java/roboy/linguistics/sentenceanalysis/SemanticParserAnalyzer.java.orig

```diff
@@ -59,6 +41,17 @@ public SemanticParserAnalyzer(int portNumber) {
      */
     @Override
     public Interpretation analyze(Interpretation interpretation) {
+<<<<<<< HEAD
```

Comment:
merge conflict

Reply:
A .orig file that slipped into the commit... I'll remove it